app/tests/issuer/test_issuer.py

```diff
@@ -234,6 +236,7 @@ async def test_request_credential_x_no_schema_cred_def(
 
     v1_record.credential_definition_id = None
     v1_record.schema_id = None
+    v1_record.type = "indy"
```

Comment:
If we set "indy" as the default type in the CredentialExchange definition, then setting it here won't be necessary

Reply:
Looks like `mock(CredentialExchange) ` does not inherit the default values of the model. Mr GPT had this to say: `No, when you use the mockito package to create a mock object, it does not automatically inherit default values from the actual Pydantic model. The purpose of creating a mock object is to simulate the behavior of the original object without involving its actual implementation, including default values.`